src/coreclr/interpreter/compiler.cpp

```diff
@@ -1364,7 +1364,7 @@ void InterpCompiler::EmitBinaryArithmeticOp(int32_t opBase)
     }
     else
     {
-#if SIZEOF_VOID_P == 8
+#if __SIZEOF_POINTER__ == 8
```

Comment:
The def used throughout code is `TARGET_64BIT`

Reply:
Ah, just wrote the same thing at the same time https://github.com/dotnet/runtime/pull/114860#discussion_r2058497659. 😅